api/v1alpha1/ext_proc_types.go

```diff
@@ -24,6 +24,18 @@ const (
 	BufferedPartialExtBodyHeaderProcessingMode ExtProcBodyProcessingMode = "BufferedPartial"
 )
 
+// +kubebuilder:validation:Enum=Route;Backend;All
+type ExtProcStage string
```

Comment:
Should we use a more common name like `ExecuteStage`? It could be used for other filters as well.